### ISSUE TITLE
Custom Character Option

### ISSUE THREAD

**Issue by megazear7** [megazear7/hackmastertrack#93]
### Suggestion:

On the character show page, have a button (with a warning message) to make the character custom. If a character is custom, the "edit" button appears, and "take" options appear on skills, talents and proficiencies. If a character is not custom, there is no take option or edit button (the take option for items would remain). Indicate the custom / not custom status on the character overview tab.
 
Thought: Rename "take" to "take for free" to communicate the meaning to the user. It might also be important to communicate why one character has this option and another does not, otherwise the user may be confused.

@BuckRogers1965 What do you think about this? I don't want to confuse the user but the functionality might be useful.

**Comment by BuckRogers1965** [comment]
We may want to let people edit a few things like handedness, name, and appearance and a few other fields without having to make the character custom. Maybe make another tab for these editable fields.

**Comment by megazear7** [comment]
What about the ability to take skills, talents and profs for free? but only for custom characters?

**Comment by megazear7** [comment]
And yeah I like that idea about being able to make some basic adjustments without being labeled custom.

**Comment by BuckRogers1965** [comment]
I do like that... once we have a dungeon master module working, the DM would be able to give stuff to the players at will, without marking them as custom.

**Comment by BuckRogers1965** [comment]
Really, the only thing we are guarding is the Build Point value, at the most basic level.  Because the player could assign themselves more silver and assign themselves more build points and simulate the take functionality.  Only they cannot edit the build points value without marking the character as custom.

**Comment by megazear7** [comment]
yeah that's true. maybe it would be better just to force them to take bps,
marking their character as custom, and the spend them. instead of creating
new, possibly confusing, functionality.

**Comment by BuckRogers1965** [comment]
Maybe we can just log when the user gives themselves more money and more bps.

**Comment by megazear7** [comment]
Ah I like that idea a lot
